code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -188,9 +188,9 @@
 	custom_metabolism = AMOUNT_PER_TIME(1, 5 SECONDS)
 	overdose = 2
 	overdose_critical = 3
-	chemclass = CHEM_CLASS_SPECIAL
+	chemclass = CHEM_CLASS_RARE
```

Comment:
Why?

Reply:
A. Prevents it from showing up in research objectives, despite being unobtainable

B. 
![image](https://github.com/user-attachments/assets/43080ceb-fcdf-427f-9fe9-383e42d01159)

Reply:
This is fine.